

---

geometry_msgs/PoseStamped goal

